pkg/id/id.go

```diff
@@ -38,6 +38,16 @@ func NewStringFromTime(time time.Time) (string, error) {
 	return id.value.String(), nil
 }
 
+// MustNewString returns an ID as a string and panics if one cannot be generated.
+func MustNewString() string {
```

Comment:
Please don't add methods that `panic`. The benefit of this is saving a line in a test, not worth it.